keyboards/idb_60/idb_60.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _IDB_60_H
+#define _IDB_60_H
+
+#include "quantum.h"
+#include "led.h"
+
+inline void _idb_60_caps_led_on(void)      { DDRC |=  (1<<5); PORTC &= ~(1<<5); }
```

Comment:
You should be able to use the [GPIO Commands](https://docs.qmk.fm/#/internals_gpio_control) for Lines 7-11:

```c
inline void _idb_60_caps_led_on(void) {
    setPinOutput(C5);
    writePinLow(C5);
}

inline void _idb_60_esc_led_on(void) {
    setPinOutput(C4);
    writePinLow(C4);
}

inline void _idb_60_caps_led_off(void) {
    setPinInput(C5);
    writePinLow(C5);
}

inline void _idb_60_esc_led_off(void) {
    setPinInput(C4);
    writePinLow(C4);
}
```